specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -335,158 +333,36 @@
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "The participants were successfully updated. The action returns an updated list of participants.",
-            "schema": {
-              "$ref": "#/definitions/ParticipantsCollection"
-            }
-          },
-          "default": {
-            "description": "Error response",
-            "schema": {
-              "$ref": "#/definitions/CommunicationErrorResponse"
-            },
-            "headers": {
-              "x-ms-error-code": {
-                "x-ms-client-name": "ErrorCode",
-                "type": "string"
-              }
-            }
-          }
-        },
-        "x-ms-examples": {
-          "Update participant": {
-            "$ref": "./examples/Rooms_UpdateParticipants.json"
-          }
-        }
-      }
-    },
-    "/rooms/{roomId}/participants:remove": {
-      "post": {
-        "tags": [
-          "Participants"
-        ],
-        "summary": "Remove participants from a room.",
-        "operationId": "Rooms_RemoveParticipants",
-        "consumes": [
-          "application/json"
-        ],
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "in": "path",
-            "name": "roomId",
-            "description": "Room id to remove the participants from.",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "$ref": "#/parameters/ApiVersionParameter"
-          },
-          {
-            "in": "body",
-            "name": "removeParticipantsRequest",
-            "description": "Participants in a room to be removed.",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/RemoveParticipantsRequest"
-            }
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "The participants were successfully deleted. The action returns an updated list of participants.",
-            "schema": {
-              "$ref": "#/definitions/ParticipantsCollection"
-            }
-          },
-          "default": {
-            "description": "Error response",
-            "schema": {
-              "$ref": "#/definitions/CommunicationErrorResponse"
-            },
-            "headers": {
-              "x-ms-error-code": {
-                "x-ms-client-name": "ErrorCode",
-                "type": "string"
-              }
-            }
-          }
-        },
-        "x-ms-examples": {
-          "Remove participant": {
-            "$ref": "./examples/Rooms_RemoveParticipants.json"
-          }
-        }
-      }
     }
   },
   "definitions": {
     "CommunicationErrorResponse": {
       "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
     },
+    "CommunicationError": {
+      "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationError"
```

Comment:
pls use the newer version of the common swagger - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/communication/data-plane/Common/stable/2022-07-13

Reply:
Updated